api/controllers/stats.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const Trials = require('../models/trial');
+const Records = require('../models/record');
+const Locations = require('../models/location');
+const Promise = require('bluebird');
+
+function getStats(req, res) {
+  let promises = [
```

Comment:
This should be `const`. Eslint should have complained about this, not sure why it haven't...